main/gradebook/lib/be/category.class.php

```diff
@@ -1950,7 +1950,7 @@ public static function register_user_certificate($category_id, $user_id)
         $userHasSkills = false;
 
         if ($skillToolEnabled) {
-            if (!$category->getGenerateCertificates()) {
+            if ($category->getGenerateCertificates()) {
```

Comment:
I cancel that request. The function is actually there to tell *if* we have configured the certificates generation or not